### DISCUSSION TITLE
Quick Question

### DISCUSSION THREAD

**Discussion by CubieJudy** [BEEmod/BEE2-items#4198] [Tech Support / Q&A] [answered]
What is the strange blue and Purple fizzler near the top left?

![38712280-27886d12-3efd-11e8-8ec1-6d9dc332098a](https://user-images.githubusercontent.com/37309250/42528798-a269f554-844a-11e8-8496-fa888a186321.png)

**Comment by TeamSpen210** [comment, marked as answer]
That was the lifeform sensor.
